### DISCUSSION TITLE
添加一个手动添加节点的按钮

### DISCUSSION THREAD

**Discussion by yyshell** [immortalwrt/homeproxy#118] [Ideas]
目前只能通过复制链接的方式添加节点，是否考虑添加一个”添加“按钮，供用户配置节点？

**Comment by d4rkblue** [comment]
有的，填上名字点新建就可以

**Reply by yyshell** [reply]
嗯，现在可以了。很奇怪的事，之前我随便填个名字，提示不支持的地址，可能有什么字符导致误判成地址了。
